datalad/plugin/wtf.py

```diff
@@ -35,12 +37,23 @@ class WTF(Interface):
             no dataset is given, an attempt is made to identify the dataset
             based on the current working directory.""",
             constraints=EnsureDataset() | EnsureNone()),
+        no_sensitive=Parameter(
+            args=("-ns", "--no-sensitive",),
```

Comment:
Not necessarily a reason not to use `-ns`, but grepping around, it looks like this would be the first spot in datalad where the short option consists of more than one letter.

Reply:
that is true!  any recommendations on what should it be otherwise?

Reply:
No good ones.  I don't like `-n` because it makes me think "dry run".  Since you didn't use it in the first place, I'm guessing you also don't like it, perhaps for the same reason.  If you renamed the long option to `--exclude-sensitive`, I think `-x` would work.  The long option would be longer, but I guess that's not a big deal when there is a short option.

Reply:
ok, will do exclude (although I do like shorter --no)